### ISSUE TITLE
Dar destaque a definições secundárias geradas num parágrafo

### ISSUE THREAD

**Issue by simas0** [livro-aberto/BookCloud#329]
Gostaria de dar às definições escritas em um parágrafo algum destaque, mesmo que isso seja alterado depois. Combinamos de usar o comando `.. glossary::` para as definições principais e o comando `:index:` para aquelas colocadas no meio de um parágrafo. A primeira já possui um destaque dado pela linguagem do sphinx, o outro não dá qualquer destaque no texto.

No código precisamos que os termos no texto criados com  
`` :index:`termo no texto <termo no glossario>` ``
sejam colocados em _itálico_.

Acho que o comando `:index:` também está sendo utilizado para levar usos de termos definidos para o glossário. Mas em princípio, eles podem receber o mesmo destaque que as definições. 

O que me diz @hjbortol ?

**Comment by hjbortol** [comment]
Sim, eu gostaria muito que as palavras marcadas com :index: tivessem uma formatação diferente. Vejo que muitos livros usam negrito, mas mesmo itálico é melhor do que formatação alguma.

**Comment by hjbortol** [comment]
Sim, eu gostaria muito que as palavras marcadas com :index: tivessem uma formatação diferente. Vejo que muitos livros usam negrito, mas mesmo itálico é melhor do que formatação alguma.

**Comment by gutosurrex** [comment]
Gente, eu procurei isso e não achei um lugar que esteja feito do jeito correto. Outras pessoas já precisaram disso:

https://stackoverflow.com/questions/46505432/inline-index-keyword-highlighted

Eles sugerem usar o método:
```
In this chapter, we introduce :index:`\ <Foo>`\ *Foo*, a
crucial concept in this example
```
que funciona, mas é claramente uma gambiarra. Mais uma vez se faz necessário aprender a extender o Sphinx.

**Comment by simas0** [comment]
@gutosurrex, eu me pergunto se esta alternativa que você propõe vale a pena. Ela não será compatível com a solução definitiva porque não será possível trocar o itálico por negrito, por exemplo, se isso for uma decisão posterior.

A solução definitiva é que coloquei no início da discussão, certo? que o `termo no texto` em 
``
:index: `termo no texto <termo no glossário>`
``
apareça em itálico.

Melhor deixar como está e procurar algo que seja realmente uma solução.

**Comment by simas0** [comment]
Este é o problema de edição TOP4.
